tests/test_node.py

```diff
@@ -58,11 +58,44 @@ def test_run(
         (master, ) = dcos_cluster.masters
         default = dcos_cluster.default_ssh_user
 
-        echo_result = master.run(args=['echo', '$USER'], user=default)
+        # Arguments are quoted - spaces and special characters are preserved
```

Comment:
Note that all of the tests in this module use a shared cluster. If the reason that you put all of these assertions into one test was to save time, you can split them up. If not, I'm OK 😄

Reply:
That is, I would probably go for `def test_shell_false` and `def test_shell_true` or similar. Maybe even `class TestShell`.